cloudify_agent/shell/commands/daemons.py

```diff
@@ -152,13 +187,14 @@ def create(**params):
     Creates and stores the daemon parameters.
 
     """
-
     attributes = dict(**params)
     custom_arg = attributes.pop('custom_options', ())
     attributes.update(_parse_custom_options(custom_arg))
     click.echo('Creating...')
-    from cloudify_agent.shell.main import get_logger
 
+    _create_rest_ssl_cert(attributes)
+    _create_broker_ssl_cert(attributes)
```

Comment:
additional params may be retrieved in get_broker_configuration in the following line?